Makefile

```diff
@@ -319,6 +319,16 @@ azure-secrets:
 	oc create secret generic codecov-token --from-literal=upload=${CODECOV_UPLOAD_TOKEN} -o yaml --dry-run | oc apply -n azure -f -
 .PHONY: azure-secrets
 
+vsphere-secrets:
+	oc create secret generic cluster-secrets-vsphere \
+	--from-file=cluster/test-deploy/vsphere/.awscred \
+	--from-file=cluster/test-deploy/vsphere/ssh-privatekey \
+	--from-file=cluster/test-deploy/vsphere/ssh-publickey \
+	--from-file=cluster/test-deploy/vsphere/pull-secret \
+	--from-file=cluster/test-deploy/vsphere/secret.tfvars \
```

Comment:
We will need this in Bitwarden

Reply:
@dav1x would you mind working with Steve on that?

Reply:
Sure. I'll ping @stevekuznetsov